flight/Modules/UAVOFrSKYSensorHubBridge/UAVOFrSKYSensorHubBridge.c

```diff
@@ -178,15 +178,15 @@ static const uint8_t frsky_rates[] = {
 // ****************
 // Private variables
 
-static struct pios_thread *uavoFrSKYSensorHubBridgeTaskHandle;
+static struct {
+	struct pios_thread *task_handle;
 
-static uint32_t frsky_port;
+	uint32_t frsky_port;
 
-static bool module_enabled;
+	uint8_t frame_ticks[MAXSTREAMS];
 
-static uint8_t *frame_ticks;
-
-static uint8_t *serial_buf;
+	uint8_t serial_buf[FRSKY_MAX_PACKET_LEN];
+} *frsky_state;
```

Comment:
I'm fine with any convention for these.  It's a pattern I'd like to start using for modules instead of globals-- a state structure with a pointer, so these things only consume 4 bytes of BSS instead of a bunch more memory.